package.json

```
@@ -8,6 +8,7 @@
     "build:ts": "tsc -b tsconfig.project.json",
     "lint:prettier": "turbo run lint:prettier",
     "check:toml": "taplo format --check",
+    "docs:dev": "turbo run dev --filter=turbo-site",
```

This doesn't match the package name in `docs/site`
```suggestion
    "docs:dev": "turbo run dev --filter=turborepo-docs",
```